ZenPacks/zenoss/Microsoft/Windows/datasources/PerfmonDataSource.py

```diff
@@ -197,8 +197,17 @@ def __init__(self, dsconf, num_commands):
     def _create_commands(self, num_commands):
         """Create initial set of commands according to the number supplied."""
         self.num_commands = num_commands
-        return [LongCommandClient(createConnectionInfo(self.dsconf))
-                for i in xrange(num_commands)]
+
+        def _long_cmd_client_wrap():
+            try:
+                conn_info = createConnectionInfo(self.dsconf)
+                return LongCommandClient(conn_info)
+            except Exception as e:
```

Comment:
Is there a way to catch specific exceptions, and probably LOG correct message for all cases?